src/ontology/mondo-edit.obo

```diff
@@ -255125,15 +255125,15 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
 intersection_of: predisposes_towards MONDO:0005579 ! epilepsy, idiopathic generalized
 relationship: excluded_subClassOf MONDO:0005579 {source="OMIM:607628", source="https://orcid.org/0000-0001-5208-3432"} ! epilepsy, idiopathic generalized
 relationship: excluded_subClassOf MONDO:0009696 {source="OMIM:607628", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile myoclonic epilepsy
-relationship: excluded_subClassOf MONDO:0011876 {source="OMIM:607628", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile absence epilepsy
+relationship: excluded_subClassOf MONDO:0800453 {source="OMIM:607628", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile absence epilepsy
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2020 {source="MONDO:mim2gene_medgen"} ! CLCN2
 relationship: predisposes_towards MONDO:0005579 {source="OMIM:607628"} ! epilepsy, idiopathic generalized
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/2543" xsd:anyURI
 
 [Term]
 id: MONDO:0011876
-name: juvenile absence epilepsy
-def: "Juvenile absence epilepsy (JAE) is a genetic epilepsy with onset occurring around puberty. JAE is characterized by sporadic occurrence of absence seizures, frequently associated with a long-life prevalence of generalized tonic-clonic seizures (GTCS) and sporadic myoclonic jerks." [Orphanet:1941]
+name: obsolete juvenile absence epilepsy
+def: "OBSOLETE. Juvenile absence epilepsy (JAE) is a genetic epilepsy with onset occurring around puberty. JAE is characterized by sporadic occurrence of absence seizures, frequently associated with a long-life prevalence of generalized tonic-clonic seizures (GTCS) and sporadic myoclonic jerks." [Orphanet:1941]
```

Comment:
if the term is split, the def should be moved to the new term

Reply:
The definition is on the new term: https://github.com/monarch-initiative/mondo/pull/7207/files#diff-889e0a9ba62d62cc6ee7c5c9bead0075444d506253590ef0dcf4786904ac6768R520233

Are you saying the definition property here should be deleted?

Reply:
yes!